src/main/java/games/strategy/triplea/ui/menubar/FileMenu.java

```diff
@@ -59,7 +59,7 @@ private JMenuItem createSaveMenu() {
     return menuFileSave;
   }
 
-  JMenuItem addPostPBEM() {
+  private JMenuItem addPostPbem() {
```

Comment:
This method was not used outside of this class, so I reduced its visibility to private when I renamed it.